arrows/pdal/pointcloud_io.h

```diff
@@ -12,25 +12,34 @@
 
 #include <vital/types/local_geo_cs.h>
```

Comment:
We should probably move `pointcloud_io.h` and `pointcloud_io.cxx` into an `algo` subdirectory under `pdal` now that these files are an implementation of an algorithm in `vital/algo`.  This is the new pattern we are moving to help keep things tidy.  See the `mvg` arrow as an example.